packages/flash-swap/test/integration/hifiFlashUniswapV2/effects/uniswapV2Call.ts

```diff
@@ -51,14 +51,14 @@ export default function shouldBehaveLikeUniswapV2Call(): void {
   context("when the caller is not the UniswapV2Pair contract", function () {
     it("reverts", async function () {
       const sender: string = this.signers.raider.address;
-      const token0Amount: BigNumber = Zero;
-      const token1Amount: BigNumber = USDC("20000");
+      const token0Amount: BigNumber = USDC("20000");
+      const token1Amount: BigNumber = Zero;
       const data: string = "0x";
       await expect(
         this.contracts.hifiFlashUniswapV2
           .connect(this.signers.raider)
           .uniswapV2Call(sender, token0Amount, token1Amount, data),
-      ).to.be.revertedWith(HifiFlashUniswapV2Errors.CallNotAuthorized);
+      ).to.be.revertedWith("");
```

Comment:
Rather than `to.be.revertedWith` we should use `to.be.reverted`.

Reply:
Another issue: this test has to be refactored to pass some dummy data. The check for `msg.sender` has moved toward the middle of the function, thus before that check is reached, the execution reverts because the data cannot be ABI decoded properly.